ui/src/containers/BasicDetails/index.tsx

```diff
@@ -35,11 +35,10 @@ import TooltipDisplay from '../../components/TooltipDisplay';
 import Rating from '../Rating';
 import { titleCase } from '../../common/titlecase';
 import { assert } from '../../store/utils';
-import { ICatalogStore } from '../../store/catalog';
 import './BasicDetails.css';
 
 const BasicDetails: React.FC = () => {
-  const { resources, catalogs }: { resources: IResourceStore; catalogs: ICatalogStore } = useMst();
+  const { resources, catalogs } = useMst();
```

Comment:
Any reason to remove the types here

Reply:
we don't need it here , as I have mentioned type in the root store